x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/index.tsx

```diff
@@ -859,6 +859,7 @@ export function Detail() {
               packageInfo={packageInfo}
               packageMetadata={packageInfoData?.metadata}
               startServices={services}
+              isCustom={shouldAllowEdit}
```

Comment:
I'm not sure about the equivalency here (perhaps in the future we disallow editing of other types of packages, besides custom ones)

I suggest to rename both the component prop and the state var to `isCustomPackage`. this will align it with the similar var `isUploadedPackage` in the other file, and the meaning will be more clear